JASP-Engine/JASP/R/ttestonesample.R

```diff
@@ -388,6 +413,15 @@ TTestOneSample <- function(jaspResults, dataset = NULL, options, ...) {
 }
 
 .ttestOneSampleDescriptivesPlotFill <- function(dataset, options, variable) {
+  errors <- .hasErrors(dataset, 
+                       message = 'short', 
+                       type = c('observations', 'variance', 'infinity'),
+                       all.target = variable,
+                       observations.amount = c('> 5000'))
```

Comment:
Ah put it on the wrong one -- this > 5000

Reply:
Yeah sorry, changed this back to `observations.amount = c('< 2')`